homeassistant/components/flexit_bacnet/sensor.py

```diff
@@ -0,0 +1,252 @@
+    FlexitSensorEntityDescription(
+        key="electric_heater_power",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.KILO_WATT,
+        translation_key="electric_heater_power",
+        value_fn=lambda data: data.electric_heater_power,
+        suggested_display_precision=3,
+    ),
+    FlexitSensorEntityDescription(
+        key="air_filter_operating_time",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=0,
+        suggested_unit_of_measurement=UnitOfTime.HOURS,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        translation_key="air_filter_operating_time",
+        value_fn=lambda data: data.air_filter_operating_time,
+        # should we have a last_reset?
+    ),
+    FlexitSensorEntityDescription(
+        key="heat_exchanger_efficiency",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key="heat_exchanger_efficiency",
+        value_fn=lambda data: data.heat_exchanger_efficiency,
+    ),
+    FlexitSensorEntityDescription(
+        key="heat_exchanger_speed",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key="heat_exchanger_speed",
+        value_fn=lambda data: data.heat_exchanger_speed,
+    ),
+)
+
+# These attributes are already in the climate entity
+#   operation_mode
+#   ventilation_mode
+
+# Comfort button state: shows true or false if the hvac is in mode HOME or AWAY, and is probably not needed as a sensor
+# room humidity 1,2,3: gives readings if extra wireless hardware is installed (so not needed)
+# If the sensor with bacnet address 59 is non existent use 95 AND 96 for in machine humidity sensors
+
+# These attributes exist in flexit_bacnet but seem too redundant as sensors since they almost never change
+#   air_temp_setpoint_away
+#   air_temp_setpoint_home
+#   fan_setpoint_supply_air_home
+#   fan_setpoint_extract_air_home
+#   fan_setpoint_supply_air_high
+#   fan_setpoint_extract_air_high
+#   fan_setpoint_supply_air_away
+#   fan_setpoint_extract_air_away
+#   fan_setpoint_supply_air_cooker
+#   fan_setpoint_extract_air_cooker
+#   fan_setpoint_supply_air_fire
+#   fan_setpoint_extract_air_fire
+#   air_filter_exchange_interval
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Flexit (bacnet) sensor from a config entry."""
+    coordinator: FlexitCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    _LOGGER.info("Setting up Flexit (bacnet) sensor from a config entry")
+
+    async_add_entities(
+        [
+            FlexitSensor(coordinator, description, config_entry.entry_id)
+            for description in SENSOR_TYPES
+        ]
+    )
+
+    # TODO: unsubscribe on remove
+
+
+class FlexitSensor(FlexitEntity, CoordinatorEntity, SensorEntity):
```

Comment:
FlexitEntity is already inheriting CoordinatorEntity, so we don't need to do that again